### DISCUSSION TITLE
AppleBleConfiguration

### DISCUSSION THREAD

**Discussion by AppGod95** [shinyorg/shiny#1399] [Q&A] [answered]
I have IBleManager injected into my service class implementation, but i would like to have access to and change the platform specific AppleBleConfiguration (in my situation, i just want to change the ShowPowerAlert bool to true, so i can have the default iOS behavior). Is it possible in any way to change this without messing too much with the implementation?

**Comment by aritchie** [comment, marked as answer]
You can do following for a MAUI single project:

```csharp
#if IOS
services.AddBluetoothLE(null, new AppleBleConfiguration(true));
#elif ANDROID
services.AddBluetoothLE();
#endif
```
